robot/OutboundFundsNPSP/tests/browser/Requirements/Requirements.robot

```diff
@@ -45,5 +45,6 @@ Add a Requirement on a Funding Request
     Click Save
     wait until modal is closed
     Click Related List Link With Text           ${req_name}
+    Select Tab                                  Details
```

Comment:
This change fixes the error in Tests/Browser/Requirements - `locator not found` by actually navigating to the Details tab (default landing tab is Related)

Reply:
🎉 !!